lms/static/scripts/frontend_apps/components/Dialog.js

```diff
@@ -55,6 +55,7 @@ export default function Dialog({
   buttons,
 }) {
   const dialogTitleId = useUniqueId('dialog');
+  const dialogDescriptionId = useUniqueId('description');
```

Comment:
```suggestion
  const dialogTitleId = useUniqueId('dialog-title');
  const dialogDescriptionId = useUniqueId('dialog-description');
```

Perhaps use a slightly more specific ID prefix here.